### ISSUE TITLE
V12 Warning message

### ISSUE THREAD

**Issue by jgordijo** [drewg13/foundryvtt-band-of-blades#47]
![image](https://github.com/user-attachments/assets/0c2787a6-4b1b-4f86-b7f7-7b64422d0230)
Foundry v12 is showing a warning for `templateVersion`.

Seems like the change is on `system.json`:

removing:
```
 {
    "templateVersion": 1
 }
```

**Comment by drewg13** [comment]
Yeah, not sure how that got in there, but it's also not hurting anything, so not worth a release.  I'll try to remember to remove it.
